flocker/control/test/test_persistence.py

```diff
@@ -123,6 +134,24 @@ def saved_again(_):
         d.addCallback(saved_again)
         return d
 
+    @validate_logging(
+        lambda test, logger:
+        test.assertEqual(len(logger.flush_tracebacks(ZeroDivisionError)), 1))
```

Comment:
Well, you'll get the line number of the test module, so doesn't seem like a problem to me.